auth0/resource_auth0_connection.go

```diff
@@ -229,7 +255,7 @@ func buildConnection(d *schema.ResourceData) *management.Connection {
 
 			if options, ok := v.(map[string]interface{}); ok {
 				c.Options = &management.ConnectionOptions{
-					Validation:                   options["validation"].(map[string]interface{}),
+					Validation:                   encodeValidation(options["validation"]),
```

Comment:
Could we follow the `buildConnection...` naming convention as used [here](https://github.com/yieldr/terraform-provider-auth0/blob/master/auth0/resource_auth0_connection.go#L290)?

It could become `buildConnectionValidation` for example.